### ISSUE TITLE
go:embed does not support all: prefix

### ISSUE THREAD

**Issue by abhinav** [bazelbuild/rules_go#3201]
### What version of rules_go are you using?

0.32.0

### What version of gazelle are you using?

0.25.0

### What version of Bazel are you using?

```
% bazel version
Build label: 5.1.0
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Thu Mar 24 14:02:30 2022 (1648130550)
Build timestamp: 1648130550
Build timestamp as int: 1648130550
```

### Does this issue reproduce with the latest releases of all the above?

Yes.

### What operating system and processor architecture are you using?

```
% uname -sm
Linux x86_64
```

### What did you do?

Attempt to use `go:embed` with a `all:` prefix on a pattern.

```
//go:embed all:templates
var templates embed.FS
```

### What did you expect to see?

Per the [documentation of the embed package](https://pkg.go.dev/embed), this should match all files inside a directory -- even those starting with with "." or "_" which are normally ignored.

> If a pattern names a directory, all files in the subtree rooted at that directory are embedded (recursively), except that files with names beginning with ‘.’ or ‘_’ are excluded. [..]
> If a pattern begins with the prefix ‘all:’, then the rule for walking directories is changed to include those files beginning with ‘.’ or ‘_’. [..]

### What did you see instead?

```
compilepkg: [snip]:11:12: could not embed all:templates: no matching files found
```

This happens whether I run `gazelle` to populate embedsrcs, or manually fill it with `glob(["templates/**"])`.

**Comment by dilyevsky** [comment]
I hit the same issue. It looks like it's at least trying to support it here https://github.com/bazelbuild/rules_go/blob/190f72c518b4a3db3c61b08a93d3715db9cb5a95/go/tools/builders/embedcfg.go#L302 but maybe there's a bug in there somewhere.

**Comment by dilyevsky** [comment]
Actually looks like it's only been implemented recently in https://github.com/bazelbuild/rules_go/pull/3455

**Comment by abhinav** [comment]
Thanks for finding that, @dilyevsky.
Closing the issue.

Resolved by #3455
